exercises/isogram/canonical-data.json

```diff
@@ -59,6 +59,14 @@
           },
           "expected": false
         },
+        {
+          "description": "word with duplicated character in camel case",
+          "property": "isIsogram",
+          "input": {
+            "phrase": "alphAbet"
```

Comment:
Incidentally, `isIsogram` is also a valid phrase for this test.

Reply:
From the description it looks like `isIsogram` would be valid test.

Are the conditions:

* single letter duplicated
* first occurrence lowercase
* second occurrence uppercase

?

Perhaps the description could be modified to better explain the problem.